lib/wrap-queries.js

```diff
@@ -19,8 +29,10 @@ function wrap() {
 
       // e.g. print.css
       if (name === 'print') {
+        hasAtRule(src);
         contents = new Buffer('\n@media print {\n' + src + '}');
       } else if (!isNaN(startingNumber)) {
+        hasAtRule(src);
```

Comment:
Yeah, tossing an error is difficult like that, because Errors/Exceptions are usually meant to be caught by another developer, i.e., not expected run-time behaviour.

Is it better to let their @ rule through, or should it disappear?